### PR TITLE
Update Versioning.md to include 1.6 opset value and add deprecated operators table

### PR DESCRIPTION
**Pull request by fdwr**
The 1.6 release missed adding a row for opset 11 to the table.

Many experimental operators were removed from ONNX, including dozens used in client models for published applications, which now cause warnings to be written to the console in `check_node`. e.g. "*Warning: Affine was a removed experimental op. In the future, we may directly reject this operator...*". These warnings provide no guidance on what to do though, and they're confusing to the user given frameworks like ONNX Runtime continue to support these operators anyway for backwards compatibility. Since this is related to versioning, I'm proposing to document the guidance here (note we can't it in Operators.md because these operators don't exist anymore, and we can't include it in ChangeLog.md either because all records of these operators were removed).

Part of #2239.

### CODE REVIEW

**Comment by fdwr** [comment]
@linkerzhang Here is the documentation aspect of the issue.

File: docs/Versioning.md [hunk 1]
```diff
@@ -127,6 +127,24 @@ Notes:
 - Values that are new or updated from a previous OpSet version are in **bold**.
 - \*: the operator is deprecated.
 
+### Operator Deprecations
+
+The following deprecated operators should be removed from models, substituted with newer operators, or decomposed:
```
Comment (prasanthpul) [inline docs/Versioning.md]: I think these are all experimental operators, right? You should clarify that these are experimental ops that have been deprecated. The concept of experimental ops is also deprecated now.

Non-experimental operators will remain in documentation even after deprecation. Example is Scatter op which was deprecated in opset 11

**Comment by prasanthpul** [comment]
can you fix the conflict?

File: docs/Versioning.md [hunk 2]
```diff
@@ -127,6 +127,24 @@ Notes:
 - Values that are new or updated from a previous OpSet version are in **bold**.
 - \*: the operator is deprecated.
 
+### Operator Deprecations
```
Comment (prasanthpul) [inline docs/Versioning.md]: Consider adding this info to https://github.com/onnx/onnx/blob/5ca0a09e6846fe70c9dda8b5034f4a1454c6d130/docs/ManagingExperimentalOps.md and linking to it instead. Experimental ops do not have any versioning and now the concept itself is deprecated, so it's odd to list them in the mainline documentation.

**Comment by fdwr** [comment]
Fixed PR here, with correct CLA-associated commits: https://github.com/onnx/onnx/pull/2555

**Pull request closed without merging**